packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,143 +24,61 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-button-border-radius;
 
-  @include bpk-border-radius-pill;
+  &--on-dark {
+    background-color: $bpk-core-primary-day;
```

Comment:
The `on-dark` chips are actually transparent. `$bpk-core-primary-day` is just the colour of the canvas in Figma 😄

Reply:
I have a small question. Suppose the background-color of the parent element of the Chip component is red, and if the background-color of the Chip component is transparent, then the Chip component will look red. Is this to be expected?😄

Reply:
yep, that's expected. We shouldn't have red backgrounds though 😄 In some cases though (like on a map) we would use the `on image` chips which have a `fill`

Reply:
Thank you for your help in solving my confusion